language/test/Language_test.rb

```diff
@@ -160,4 +161,17 @@
     end
   end
 
+  describe "#GetLocaleString" do
+    context "when the language is defined" do
+      it "returns the full language identifier" do
+        expect(subject.GetLocaleString("de_ZU")).to eq("de_ZU.UTF-8")
+      end
+    end
+
+    context "when the language is not defined" do
+      it "returns the full language identifier" do
+        expect(subject.GetLocaleString("ma_MA")).to eq("ma_MA.UTF-8")
+      end
+    end
+  end
```

Comment:
In these tests it not clear enough (to me, at first sight) where the `UTF-8` part comes from or if there is a possibility to add a different suffix or no suffix at all. Just looking at this, it looks like a method to append `.UTF-8` to whatever string it receives as parameter.